apps/root/src/state/all-strategies-filters/actions.ts

```diff
@@ -0,0 +1,14 @@
+import { createAction } from '@reduxjs/toolkit';
+import { ChainId, StrategyYieldType, Token } from 'common-types';
+
+export const setAssetFilter = createAction<Token[]>('allStrategiesFilters/setAssetFilter');
+
+export const setNetworkFilter = createAction<ChainId[]>('allStrategiesFilters/setNetworkFilter');
+
+export const setRewardFilter = createAction<Token[]>('allStrategiesFilters/setRewardFilter');
+
+export const setFarmFilter = createAction<string[]>('allStrategiesFilters/setFarmFilter');
```

Comment:
oh we could create a FarmId type or GuardianId type also, so we start moving away from primitives